tensornetwork/backends/jax/jax_backend.py

```diff
@@ -72,6 +72,17 @@ def cmplx_randn(complex_dtype, real_dtype):
 
     return self.jax.random.normal(key, shape).astype(dtype)
 
+  def eigs(self,
+           A: Callable,
+           initial_state: Optional[Tensor] = None,
+           ncv: Optional[int] = 200,
+           numeig: Optional[int] = 1,
+           tol: Optional[float] = 1E-8,
+           which: Optional[Text] = 'LR',
+           maxiter: Optional[int] = None,
+           dtype: Optional[Type] = None) -> Tuple[List, List]:
+    raise NotImplementedError()
```

Comment:
Don't override a not implemented as it looses the nice error message from the base.